providers/implementations/digests/blake2_prov.c

```diff
@@ -12,9 +12,6 @@
 #include "prov/digestcommon.h"
 #include "prov/implementations.h"
 
-OSSL_FUNC_digest_init_fn ossl_blake2s256_init;
```

Comment:
these were here to make sure the function signature was correct.

Reply:
Okay, back now.

Reply:
Correct that, they aren't `OSSL_FUNC_digest_init_fn` functions.  They are called by such from the macros, by that point the OSSL_PARAM arugment has been processed and isn't passed to them.